lib/bridge/github/issues.rb

```diff
@@ -150,21 +153,21 @@ def activities
       def patch(hash)
         hash["labels"] = hash["labels"].map {|l| l["name"] } if hash["labels"]
         updated = client.patch hash
-        updated.extend(Card).merge!(:repo => repo)
+        updated.extend(Card).merge!(:repo => self[:repo])
```

Comment:
no more hashie, hashie would catch method missing and pull the repo key out of its own hash